src/pages/frontpage/FrontPage.tsx

```diff
@@ -1,55 +1,26 @@
-import { Box, Typography } from '@mui/material';
-import { useTranslation } from 'react-i18next';
+import { Box, Container } from '@mui/material';
+import { FrontPageHeading } from './FrontPageHeading';
 
-const FrontPage = () => {
-  const { t } = useTranslation();
-
-  return (
-    <Box
+const FrontPage = () => (
+  <Box
+    sx={{
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      bgcolor: '#EFEFEF',
+      width: '100%',
+      height: '100vh',
+    }}>
```

Comment:
Må ta vare på denne om vi ønsker at det skal være annen bakgrunnsfarge enn ellers, men flex-en trengs vel ikke?
```suggestion
    sx={{ bgcolor: '#EFEFEF', width: '100%', height: '100vh' }}>
```